core/src/main/scala/com.snowplowanalytics.stream/loader/Emitter.scala

```diff
@@ -167,10 +167,9 @@ class Emitter(
   override def fail(records: JList[EmitterJsonInput]): Unit = {
     records.asScala.foreach {
       case (r: String, Validated.Invalid(fs)) =>
```

Comment:
Do we need this `: String`?

Reply:
We don't, I don't know why it was there. Removing it.